interp/iostream.go

```diff
@@ -0,0 +1,203 @@
+package interp
+
+import (
+	"bufio"
+	"io"
+	"os/exec"
+	"syscall"
+)
+
+const (
+	NOT_CLOSED_CODE = -127
```

Comment:
The capital letter makes this exported as `interp.NOT_CLOSED_CODE`. In addition, in Go people don't normally use `ALL_CAPS` for constants these days. I think we should rename to `notClosedExitCode`.